common/services/prismic/events.js

```diff
@@ -87,25 +98,29 @@ function parseEventDoc(document: PrismicDocument): UiEvent {
   };
 }
 
+const fetchLinks = [].concat(
+  eventAccessOptionsFields,
+  teamsFields,
+  eventFormatsFields,
+  placesFields,
+  interpretationTypesFields,
+  audiencesFields,
+  eventSeriesFields,
+  organisationsFields,
+  peopleFields,
+  contributorsFields,
+  eventSeriesFields
+);
+
 export async function getEvent(req: Request, id: string): Promise<?UiEvent> {
   const document = await getDocument(req, id, {
-    fetchLinks: [].concat(
-      eventAccessOptionsFields,
-      teamsFields,
-      eventFormatsFields,
-      placesFields,
-      interpretationTypesFields,
-      audiencesFields,
-      eventSeriesFields,
-      organisationsFields,
-      peopleFields,
-      contributorsFields,
-      eventSeriesFields
-    )
+    fetchLinks: fetchLinks
   });
 
   if (document && document.type === 'events') {
-    const event = parseEventDoc(document);
+    const scheduleIds = document.data.schedule.map(event => event.event.id);
+    const eventScheduleDocs = scheduleIds.length > 0 && await getTypeByIds(req, ['events'], scheduleIds, {fetchLinks});
+    const event = parseEventDoc(document, eventScheduleDocs || {});
```

Comment:
If we type it as `PrismicApiSearchResponse` - we could make it nullable to make sure we keep the shape.